spacecmd/src/spacecmd/schedule.py

```diff
@@ -466,7 +466,10 @@ def do_schedule_deletearchived(self, args):
     This method removes all of the archived actions older than provided date.
     If no date is provided it will delete all archived actions.
     """
-    args = args.split() or []
+    arg_parser = get_argument_parser()
+    arg_parser.add_argument('-y', '--yes', default=False, action="store_true")
```

Comment:
We should add a clause to the help text for this parameter (also for `archivecompleted`), like below:

https://github.com/uyuni-project/uyuni/blob/18c749cab559c0c2056ee728b78dd2dc2a15d4f8/spacecmd/src/spacecmd/repo.py#L255-L262

Reply:
Definetely yes. Thanks for the point :+1: